assets/src/admin/paired-browsing/app.css

```diff
@@ -0,0 +1,207 @@
+html,
+body {
+	margin: 0;
+	padding: 0;
+	height: 100%;
+	width: 100%;
+}
+
+body {
+	display: flex;
+	flex-direction: column;
+}
+
+body * {
+	font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;
+}
+
+#header {
+	height: 32px;
+	background-color: #0075c2;
+}
+
+#header * {
+	color: #fff;
+}
+
+#header ul {
+	display: flex;
+	margin: 0;
+	padding: 0;
+	list-style: none;
+	height: 100%;
+}
+
+#header li {
+	align-items: center;
+	display: flex;
+	line-height: 1.5;
+}
+
+#header-logo {
+	position: absolute;
+	left: 20px;
+	height: 32px;
+}
+
+#header-logo img {
+	display: block;
+	width: 20px;
+	height: 20px;
+	margin-right: 0.5rem;
+}
+
+#header li:last-child {
+	position: absolute;
+	right: 20px;
+	height: 32px;
+}
+
+#header .iframe-label {
+	margin: 0 auto;
+	font-weight: 600;
+	width: 50%;
+	justify-content: center;
+	text-decoration: underline;
```

Comment:
The underline here is making me think it is a link when it is not. Is this needed?